02_configure_host.sh

```diff
@@ -39,6 +39,8 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
     -e "num_masters=$NUM_MASTERS" \
     -e "num_workers=$NUM_WORKERS" \
     -e "extradisks=$VM_EXTRADISKS" \
+    -e "libvirt_nic_model=e1000" \
```

Comment:
Why change to e1000? I think once we get snponly.efi built, I got it working with the virtio network driver.

Reply:
ok, I'll try it out today